src/utils.lua

```diff
@@ -7,17 +7,6 @@ function utils.hasMatchingTag(tag, value)
 	return Handlers.utils.hasMatchingTag(tag, value)
 end
 
--- Then, check for a 43-character base64url pattern.
--- The pattern checks for a string of length 43 containing alphanumeric characters, hyphens, or underscores.
-function utils.isValidBase64Url(url)
-	local isValidBase64Url = #url == 43 and string.match(url, "^[%w-_]+$") ~= nil
-
-	if not isValidBase64Url then
-		error("String pattern is invalid.")
-	end
-	return url
-end
```

Comment:
cruft from aos copy/pasta - we use more specific utils and this has been confused with those